deps.edn

```diff
@@ -295,6 +295,13 @@
   :oss-dev
   {}
 
+  ;; Generate BE documentation with
+  ;; clojure -M:marginalia
+  :marginalia
+  {:extra-deps
+   {com.github.tsmacdonald/marginalia {:mvn/version "0.9.2"}}
+   :main-opts ["-m" "marginalia.main" "-n" "Metabase" "-d" "backend-docs" "-D" "The simplest, fastest way to get business intelligence and analytics to everyone in your company 😋" "dev" "src"]}
```

Comment:
You probably also want to generate dox for `shared` and `enterprise/backend/src`

Reply:
Oh snap, good call